### PR TITLE
Mumble integration

### PR DESCRIPTION
**Pull request by pepperkick**

### CODE REVIEW

File: src/server/services/pickup/hooks.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-import createMumbleChannel from './create-mumble-channel';
-import removeMumbleChannel from './remove-mumble-channel';
 
 export default {
   after: {
     create(props) {
-      createMumbleChannel(props);
       configureServer(props);
 
+      props.app.service('mumble').create(props.status.region, props.status.id);
```
Comment (HenriBeck) [inline src/server/services/pickup/hooks.js]: props.status.region is not a thing, I guess you mean props.result.region?

Comment (pepperkick) [reply src/server/services/pickup/hooks.js]: fml, Idk why I typed status

File: src/server/services/mumble-channels/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          connection.authenticate(mumbleUsername, mumblePassword);
+
+          connection.on('initialized', () => {
+            this.connections[region.name] = connection;
+          });
+        });
+      });
+  }
+
+  create({
```
Comment (HenriBeck) [inline src/server/services/mumble-channels/index.js]: This should be async and wait for the channel creation

File: src/server/services/mumble-channels/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    region,
+    name,
+  }) {
+    const channel = this.connections[region].channelByName('Pickups');
+
+    channel.addSubChannel(name);
+  }
+
+  delete({
```
Comment (HenriBeck) [inline src/server/services/mumble-channels/index.js]: Same with this method

File: src/server/services/index.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ import pickup from './pickup';
 import pickupQueue from './pickup-queue';
 import servers from './servers';
 import slack from './slack';
+import mumble from './mumble-channels';
```
Comment (HenriBeck) [inline src/server/services/index.js]: Rename to mumbleChannels

File: src/server/services/mumble-channels/index.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+   * @param {Object} channel.region - Mumble channel region.
+   * @param {Object} channel.name - Mumble channel name.
+   */
+  async create({
+    region,
+    name,
+  }) {
+    const channel = await this.connections[region].channelByName('Pickups');
+
+    await channel.addSubChannel(name);
```
Comment (HenriBeck) [inline src/server/services/mumble-channels/index.js]: Are those method's sync or async?

File: src/server/services/mumble-channels/index.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    const mumbleUsername = config.get('server.mumble.username');
+    const mumblePassword = config.get('server.mumble.password');
+
+    /**
+     * Handle mumble connection.
+     *
+     * @param {String} region - Mumble region.
+     * @param {Object} connection - Mumble connection.
+     */
+    const handleConnection = (region, connection) => {
```
Comment (HenriBeck) [inline src/server/services/mumble-channels/index.js]: This method can be moved into the class body

**Pull request merged**